db.py

```diff
@@ -0,0 +1,111 @@
+"""
+This module handles the management of the database.
+"""
+import enum
+import os
+from typing import List
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+from google.cloud.firestore_v1 import base_query
+import gradio as gr
+
+from credentials import get_credentials_json
+
+
+def get_environment_variable(name: str) -> str:
+  value = os.getenv(name)
+  if value is None:
+    raise ValueError(f"Environment variable {name} is not set")
+  return value
+
+
+RATINGS_COLLECTION = get_environment_variable("RATINGS_COLLECTION")
+SUMMARIZATIONS_COLLECTION = get_environment_variable(
+    "SUMMARIZATIONS_COLLECTION")
+TRANSLATIONS_COLLECTION = get_environment_variable("TRANSLATIONS_COLLECTION")
+
+if gr.NO_RELOAD:
+  firebase_admin.initialize_app(credentials.Certificate(get_credentials_json()))
+  db = firestore.client()
+
+
+class Category(enum.Enum):
+  SUMMARIZATION = "summarization"
+  TRANSLATION = "translation"
+
+
+class Rating:
+
+  def __init__(self, model: str, rating: int):
+    self.model = model
+    self.rating = rating
```

Comment:
You can use a dataclass for it. You only need to define:
```python
@dataclasses.dataclass
class Rating:
  model: str
  rating: int
```
And the constructor will be created automatically.
I think a dataclass is much more readable.